bc_obps/registration/api/v2/_operations/_operation_id/bcghg_id.py

```diff
@@ -0,0 +1,27 @@
+from typing import Literal, Tuple
+from uuid import UUID
+from django.http import HttpRequest
+from registration.schema.v2.operation import OperationBoroIdOut
+from service.operation_service_v2 import OperationServiceV2
+from registration.constants import V2
+from common.permissions import authorize
+from registration.api.utils.current_user_utils import get_current_user_guid
+from service.error_service.custom_codes_4xx import custom_codes_4xx
+from registration.decorators import handle_http_errors
+from registration.api.router import router
+from registration.schema.generic import Message
+
+
+@router.patch(
```

Comment:
I could have made just one BCGHG ID endpoint because the only difference between operation and facility endpoints is what service to call, but keeping them separate seemed to better match our API structure and conventions